rs/src/parse/expressions/literals.rs

```diff
@@ -38,8 +38,13 @@ enum LiteralValue {
     /// May be used only for binary and FixedBinary.
     Binary(Vec<u8>),
 
-    /// May be used only for IntervalYearToMonth and IntervalDayToSecond.
-    Interval(i64, i64),
```

Comment:
Should we reject use of Interval as deprecated at this point or just return an error as we would likely do once the old fields are removed?

Reply:
I'm still new to this code base, but I think this is an internal class that is not exported; it does not have a 1:1 match with the public protobuf files. So there isn't a way to do `Interval` (and not `IntervalYearToMonth` and `IntervalDayToSecond`) in protobufs right now - there's nothing to deprecate or return an error on.

Does that make sense? (Or am I missing something?)